workspace.json

```diff
@@ -375,6 +375,31 @@
         }
       }
     },
+    "bulldozer-application-ui-dark-theme": {
+      "projectType": "library",
+      "root": "libs/bulldozer/application/ui/dark-theme",
+      "sourceRoot": "libs/bulldozer/application/ui/dark-theme/src",
+      "prefix": "heavy-duty",
```

Comment:
Every bulldozer library has the prefix of "bd" that stands for "bulldozer" rather than "heavy-duty"